### PR TITLE
Bug fixes for plots (#3019)

### PR DESCRIPTION
**Pull request by deeptailor**
* prevent plots from breaking when more than one NaN is received. fix y axis label issue

* dont emit a viewport change event when marquee doesnt happen

### CODE REVIEW

**Comment by davetsay** [comment]
@deeptailor , looks good. Please add an author checklist.

File: src/plugins/plot/src/configuration/PlotSeries.js [hunk 1]
```diff
@@ -322,7 +322,15 @@ define([
          *                  a point to the end without dupe checking.
          */
         add: function (point, appendOnly) {
-            var insertIndex = this.data.length;
+            var insertIndex = this.data.length,
+                currentYVal = this.getYVal(point),
+                lastYVal = this.getYVal(this.data[insertIndex - 1]);
+
+            if (this.isValueInvalid(currentYVal) && this.isValueInvalid(lastYVal)) {
+                console.warn('[Plot] Invalid Y Values detected');
```
Comment (nikhilmandlik) [inline src/plugins/plot/src/configuration/PlotSeries.js:330]: May be use openmct notifications or remove?

Comment (deeptailor) [reply src/plugins/plot/src/configuration/PlotSeries.js:330]: I didnt want to bombard the user, because they could be 100's of potential invalid values

Comment (deeptailor) [reply src/plugins/plot/src/configuration/PlotSeries.js:330]: Still need a message, so i decided to go via console

**Comment by deeptailor** [comment]
## Author Checklist
Changes address original issue? Yes
Unit tests included and/or updated with changes? N/A
Command line build passes? Yes
Changes have been smoke-tested? Yes
Testing instructions included? No, because its complicated to product invalid values for testing

**Pull request merged**